.github/workflows/ubuntu.yml

```diff
@@ -42,5 +42,12 @@ jobs:
       env:
         GH_TOKEN: ${{ github.token }}
     - name: Run test
+      # Tests don't pass yet on truffleruby
+      if: >-
+        matrix.ruby != 'truffleruby-head'
```

Comment:
We could also use `continue-on-error` like for jruby just below, not sure what is best.

Reply:
Right. I don't have an opinion for this. You can choose whichever you like.